net/scripts/mount_additional_disk.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
net/scripts/mount_additional_disk.sh  -- we kabab files here: `mount-additional-disk.sh`